app/src/main/java/com/ngadt/appmusicplayer/presenter/MainPresenter.kt

```diff
@@ -0,0 +1,27 @@
+import android.content.Context
+import com.ngadt.appmusicplayer.repository.MusicRepository
+import com.ngadt.appmusicplayer.repository.local.OnResultListener
+import java.lang.Exception
+
+class MainPresenter(
+    private val view: MainActivityContract.View,
+    private val musicRepository: MusicRepository
+) : MainActivityContract.Presenter {
+    override fun getLocalSongs(context: Context) {
```

Comment:
```suggestion
) : MainActivityContract.Presenter {

    override fun getLocalSongs(context: Context) {
```
chú ý cái này cho a nhé , cmt nhiều đoạn naỳ rồi